actionview/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add the template digest to the cache key for collection caching.
```

Comment:
:+1:, but I still want to add that this happened if you rendered in the controller, so maybe adding something like:


```
available from the view context. Which could happen if the rendering was done directly in the controller and not in a template.
```